scripts/v8-compiler/clang-wrap.sh

```diff
@@ -8,7 +8,7 @@
 compiler=$1
 shift
 
-out=()
+out=(-mno-sse2 -mno-sse3 -mno-sse4.1 -mno-sse4.2 -mno-sse4.1 -mno-avx -mno-avx2  -mno-avx512bf16 -mno-avx512bitalg -mno-avx512bw -mno-avx512cd -msgx)
```

Comment:
These likely aren't necessary, I'm just trying to cut everything that's not necessary